web/karma.conf.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function (config) {
+  config.set({
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://www.npmjs.com/search?q=keywords:karma-adapter
+    frameworks: ['jasmine'],
```

Comment:
A question just for future reference. Previously we seemed to use 'mocha' and 'chai' for the framework in the web SDK. Is there any specific reason to choose 'jasmine'?

Reply:
@jnmt 

This is a very good question.

This is my understanding.

Mocha is a JS test runner runtime but it doesn't contain the assertion library, so it needs to work with an assertion library. Chai is that assert library in "scalardl-web-client-sdk".

Jasmine is a test runner and also an assertion library.

In short, Jasmine is a more complete test framework.

I replace Mocha + Chai with Jasmine since I think it can reduce some dependencies.

I hope this could answer to your question 🙇

Reply:
Thank you! I understood well.